hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/LinearRetryPolicy.java

```diff
@@ -0,0 +1,157 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.fs.azurebfs.services;
+
+import org.apache.hadoop.fs.azurebfs.AbfsConfiguration;
+import org.apache.hadoop.classification.VisibleForTesting;
+
+/**
+ * Linear Retry policy used by AbfsClient.
+ * */
+public class LinearRetryPolicy extends RetryPolicy{
+  
+  /**
+   * Represents the default maximum amount of time used when calculating the
+   * linear delay between retries.
+   */
+  private static final int DEFAULT_MAX_BACKOFF = 1000 * 30; // 30s
+
+  /**
+   * Represents the default minimum amount of time used when calculating the
+   * linear delay between retries.
+   */
+  private static final int DEFAULT_MIN_BACKOFF = 500 * 1; // 500ms
+
+  /**
+   * Represents the delta by which retry interval should be incremented
+   * for each retry count
+   */
+  private static final int INTERVAL_DELTA_ONE_SEND = 1000; // 1s
```

Comment:
nit: lets have it `INTERVAL_DELTA_ONE_SECOND`

Reply:
Done